apps/heft/src/cli/actions/DevDeployAction.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { HeftActionBase, IHeftActionBaseOptions, ActionHooksBase, IActionDataBase } from './HeftActionBase';
+
+/**
+ * @public
+ */
+export interface IDevDeployActionData extends IActionDataBase<DevDeployHooks> {}
+
+/**
+ * @public
+ */
+export class DevDeployHooks extends ActionHooksBase {}
+
+export class DevDeployAction extends HeftActionBase<IDevDeployActionData, DevDeployHooks> {
+  public constructor(options: IHeftActionBaseOptions) {
+    super(
+      {
+        actionName: 'dev-deploy',
```

Comment:
This is a local jargon that I haven't seen widely used elsewhere. The name is maybe not very intuitive. (What does "dev" mean? How does it different from a normal deployment? etc.)

Reply:
Let's leave this as-is for now and potentially rename to "deploy" in the future.